IMP-/Small_StepT.thy

```diff
@@ -0,0 +1,179 @@
+ (*
+    Authors: Bilel Ghorbel, Florian Kessler
+    Based on the Big step semantics definition  with time of IMP
+ *)
+section "Small step semantics of IMP- "
+
+subsection "Small step semantics definition"
+theory Small_StepT  imports Main Big_StepT Rel_pow begin
```

Comment:
the small step semantics feels more natural than the bigstep semantics now. 
Maybe Bigstep should import smallstep, as it is now modeled such that it is equivalent to smallstep (e.g. assign costs 2).
and equivalence proof should be at the end of Bigstep or in an extra theory.

maybe even rename the theory names to `"Big_Step_Time"` , and Small_Step_Time, ( or even `IMP-_Big_Step_Time` but I don't like the IMP- )

Reply:
what about making the equivalence in a 3rd theory that imports both ? too much ?

Reply:
I moved it into a new theory now